<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_AutoRebuild</class>
 <widget class="QDialog" name="Dialog_AutoRebuild">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>201</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>计算与匹配特征点参数设置：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>特征点算子：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>特征点质量：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>计算特征点空间向量：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>特征暴力计算：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>几何模型：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>距离比率：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>暴力匹配：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>137</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>三维重建参数设置：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>SFM重建引擎类型：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>是否鲁棒化：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="comboBox_describer">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>189</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>SIFT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SIFT_ANATOMY</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AKAZE_FLOAT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AKAZE_MLDB</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_geometricmodel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>281</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>fundamental matrix</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>essential matrix</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>homography matrix</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>essential matrix with an angular parametrization</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>upright essential matrix with an angular parametrization</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>orthographic essential matrix</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_forcecompute">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>否</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>是</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_quality">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>NORMAL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HIGH</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ULTRA</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_upright">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>否</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>是</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_distanceRatio">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>0.8</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_forcematch">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>否</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>是</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_sfmEngine">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>50</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>INCREMENTAL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>INCREMENTALV2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GLOBAL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>STELLAR</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_isRobustTriangulation">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>on</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>off</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_CANCEL">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>220</y>
     <width>80</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>140</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>结果输出路径:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_browseOutputDir">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>150</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_OutputDir">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>150</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_CONFIRM">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>220</y>
     <width>80</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
